Neuro/Pathfinding/PathfindingHandler.cs

```diff
@@ -2,28 +2,62 @@
 using System.Collections.Generic;
 using System.Linq;
 using Neuro.Pathfinding.DataStructures;
+using Reactor.Utilities.Extensions;
 using UnityEngine;
 
 namespace Neuro.Pathfinding;
 
```

Comment:
This task might still be out of reach
![image](https://user-images.githubusercontent.com/32238504/230753080-fc155a37-085b-4093-b198-325eda1268f2.png)